deepchem/feat/tests/test_mat_featurizer.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from deepchem.feat import MATFeaturizer
+import numpy as np
+
+
+class TestMATFeaturizer(unittest.TestCase):
+  """
+    Test MATFeaturizer.
```

Comment:
Some formatting issues here and in other docstrings

Reply:
The formatting issues still persist (the triple quotes need to be aligned). But I wonder if `yaml` is responsible for this. I have seen this happen when I used to format too.

Reply:
Fixed! Thank you.